src/NServiceBus.AcceptanceTesting/ScenarioContext.cs

```diff
@@ -46,5 +47,17 @@ public override string ToString()
                 return $"{Level}: {Message}";
             }
         }
+
+        static readonly AsyncLocal<ScenarioContext> asyncContext = new AsyncLocal<ScenarioContext>();
+
+        internal static ScenarioContext GetContext()
```

Comment:
Can we make these a static `Current` property instead?

That would also mean this should be moved up to the top of the file.